.nvmrc

```diff
@@ -1,2 +1,2 @@
-v12.20.1
+v16
```

Comment:
Upgrade to v16 is because the new Jest version requires some new supported grammar. But these updated dependencies do not require 16, so I don't add `engine` in the package.json.

Reply:
Maybe go straight to 18?